editor/plugins/texture_region_editor_plugin.cpp

```diff
@@ -234,6 +241,14 @@ void TextureRegionEditor::_region_draw() {
 
 	updating_scroll = false;
 
+	if (request_center && hscroll->get_min() < 0) {
+		hscroll->set_value((hscroll->get_min() + hscroll->get_max() - hscroll->get_page()) / 2);
+		vscroll->set_value((vscroll->get_min() + vscroll->get_max() - vscroll->get_page()) / 2);
+		// This ensures that the view is updated correctly.
+		callable_bind(callable_mp(this, &TextureRegionEditor::_pan_callback), Vector2(1, 0)).call_deferred(nullptr, 0);
```

Comment:
I can't believe centering on the texture was so difficult. I resorted to this hack, anything better would require some bigger rewrite of the scroll logic.

One of the problems was that the scrollbars are initialized to their final value after a second update, hence the `hscroll->get_min() < 0` part.